tests/third_party/cupy/manipulation_tests/test_join.py

```diff
@@ -113,8 +113,7 @@ def test_concatenate_large_f_contiguous(self, xp, dtype):
         e = testing.shaped_arange((2, 3, 2), xp, dtype)
         return xp.concatenate((a, b, c, d, e) * 2, axis=-1)
 
-    @pytest.mark.skip("TODO: remove once dpctl #1325 is resolved")
-    @testing.numpy_cupy_array_equal()
+    @testing.numpy_cupy_array_equal(type_check=has_support_aspect64())
     def test_concatenate_many_multi_dtype(self, xp):
```

Comment:
Does it pass for you on `cpu`?
I get `Fatal Python error`
tests/third_party/cupy/manipulation_tests/test_join.py::TestJoin::test_concatenate_many_multi_dtype Fatal Python error: Aborted